src/components/ebay-progress-spinner/index.marko

```diff
@@ -0,0 +1,8 @@
+import processHtmlAttributes from "../../common/html-attributes"
+
+static var ignoredAttributes = [
+    "a11yText",
```

Comment:
Add `size` and `class` as well. Basically anything you are using as `input` should be here.